docs/rfcs/component-universal-telemetry.md

```diff
@@ -0,0 +1,118 @@
+# Auto-Instrumented Component Telemetry
```

Comment:
Not sure if this rfcs should do that or not, but would be good to have a "generic" section for components telemetry where we say what are the "recommended" attributes that everyone should use for all the signals. I know we enforce that for logger, but we don't do that for trcaes/metrics.

Reply:
I was thinking about this too. The attributes are almost a separate proposal because they apply more broadly. Really all telemetry emitted by a component should have these attributes, but this proposal only describes telemetry that can be captured automatically.

I think https://github.com/open-telemetry/opentelemetry-collector/pull/11344 demonstrates how these exact attributes can be available when every component is created. Maybe we can also pack them into the `TelemetrySettings` in a way where any signals automatically include them?

In any case, do you think I should peel off a separate RFC for attributes? (We do already have #11179 also.)

Reply:
I think we resolved this on yesterday's stability meeting, can this be marked as solved (and we can come back to this in the next revision of the RFC), or is there something left to discuss on this thread?